src/utils.js

```diff
@@ -67,14 +67,35 @@ const prepGitRepo = async () => {
   await exec(`git config --unset "http.https://github.com/.extraheader"`);
 };
 
-const setupDVC = async opts => {
+const commandExists = async command => {
   const { platform } = process;
+  try {
+    const cmd = platform === 'win32' ? `where ${command}` : `which ${command}`;
+    await exec(cmd);
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+const pipInstall = async version => {
+  const pkg = `dvc[all]${version === 'latest' ? '' : `==${version}`}`;
+  const hasUv = await commandExists('uv');
+  const opts = hasUv ? '--system' : '';
```

Comment:
Here's the comparison between using `uv` and `pip` for installing `dvc[all]==3.61.0`.

| System | pip | uv |
|--------|-----|----|
| windows-latest | [2m 12s](https://github.com/iterative/setup-dvc/actions/runs/16927952213/job/47967348458?pr=70#step:8:5) | [35s](https://github.com/iterative/setup-dvc/actions/runs/16927952213/job/47967348473?pr=70#step:8:9) |
| ubuntu-22.04-arm | [43s](https://github.com/iterative/setup-dvc/actions/runs/16927952213/job/47967348472?pr=70#step:8:12) | [2s](https://github.com/iterative/setup-dvc/actions/runs/16927952213/job/47967348480?pr=70#step:8:13) |